test/unit/kernel/src/KernelLambda.cpp

```diff
@@ -19,6 +19,10 @@
  * If not, see <http://www.gnu.org/licenses/>.
  */
 
+// NVCC needs --expt-extended-lambda
+#if !defined(__NVCC__) || \
+    ( defined(__NVCC__) && defined(__CUDACC_EXTENDED_LAMBDA__) )
```

Comment:
Could you please use `BOOST_COMP_NVCC` instead of `__NVCC__` for consistency? You would have to move the if somewhere under `#include <alpaka/core/BoostPredef.hpp>`. Maybe above `BOOST_AUTO_TEST_SUITE(kernel)`?

Reply:
tbh, I wanted to avoid fiddling with boost headers for an otherwise empty file.